python/ray/reporter.py

```diff
@@ -107,6 +112,23 @@ def __init__(self, redis_address, redis_password=None):
     def get_cpu_percent():
         return psutil.cpu_percent()
 
+    @staticmethod
+    def get_gpu_usage():
+        if gpustat is not None:
+            gpu_utilizations = []
+            gpus = []
+            try:
+                gpus = gpustat.new_query().gpus
+            except Exception:
+                logger.debug("gpustat failed to retrieve GPU information.")
```

Comment:
Why not printing the exception messages?

Reply:
Good point. I will.